addons/binding/org.openhab.binding.solarlog/src/main/java/org/openhab/binding/solarlog/SolarLogBindingConstants.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.solarlog;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SolarLogBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Johann Richard - Initial contribution
+ */
+public class SolarLogBindingConstants {
+
+    public static final String BINDING_ID = "solarlog";
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059